wurst/objects/abilities/beastMaster/SummonBeast.wurst

```diff
@@ -0,0 +1,90 @@
+package SummonBeast
+
+// Standard Library Imports:
+import AbilityObjEditing
+import ObjectIds
+import Assets
+
+// Local Imports:
+import LocalObjectIDs
+import ElkDefinition
+import JungleWolfDefinition
+import JungleBearDefinition
+
+let ICON = Icons.bTNStag
+let TOOLTIP = "Summon a beast to fight along side you"
+let TOOLTIP_EXT = "Each level will increase the ferocity of the animal fighting along your side"
+
+let DAMAGE_BASE = 7
+let DAMAGE_RANGE = 3
+let DAMAGE_PER_LEVEL = 3
+let COOLDOWN = 60.
+let DURATION = 30.
+
+class ElkLocust extends AdultElkAttackerDefinition
+    construct(int unitId)
+        super(unitId)
+        this
+            ..setName("Elk")
+            ..setNormalAbilities("Aloc")
+            ..setAttack1DamageSidesperDie(DAMAGE_RANGE)
+            ..setAttack1DamageBase(DAMAGE_BASE)
+            ..setSpeedBase(522)
+            ..setTurnRate(0.3)
+
+class WolfLocust extends AdultJungleWolfDefinition
```

Comment:
I think these sort of constructs should be dropped in favor of chain function calls relating to `AbilityDefinition` but this is OK for now.